packages/wallet-sdk/src/index.ts

```diff
@@ -13,6 +14,7 @@ declare global {
     CoinbaseWalletProvider: typeof CoinbaseWalletProvider;
     ethereum?: ProviderInterface;
     coinbaseWalletExtension?: ProviderInterface;
+    coinbaseWalletExtensionSigner?: Signer;
```

Comment:
my reason to use `coinbaseWalletExtensionSigner` instead of what we talked before, "coinbaseWalletInjectedSigner", is i think it's always better to have more granularity. cc: @bangtoven

Reply:
makes sense to me

Reply:
That makes sense to me also.